torch_frame/utils/split.py

```diff
@@ -5,25 +5,37 @@
 
 
 def generate_random_split(length: int, seed: int, train_ratio: float = 0.8,
-                          val_ratio: float = 0.1) -> np.ndarray:
+                          val_ratio: float = 0.1,
+                          include_test: bool = True) -> np.ndarray:
     r"""Generate a list of random split assignments of the specified length.
     The elements are either :obj:`0`, :obj:`1`, or :obj:`2`, representing
     train, val, test, respectively. Note that this function relies on the fact
     that numpy's shuffle is consistent across versions, which has been
     historically the case.
     """
-    assert train_ratio + val_ratio < 1
+    if include_test:
+        assert train_ratio + val_ratio < 1
+    else:
+        assert train_ratio < 1
```

Comment:
Should be `assert train_ratio + val_ratio == 1`?

Reply:
I was afraid that there maybe some numerical error. But will use math.isclose to check that.

Reply:
I don't think you need to worry about that. People will pass something like `0.9 + 0.1`

Reply:
ok